<launch>

<!-- Start Navigation -->
 <include file="$(find swot_navigation)/launch/navigation.launch"/>


<!-- Start the swot_robocup_navigation File-->
<node pkg="swot_navigation" type="swot_navigation_node" name="swot_robocup_navigation" respawn="true" output="screen"> 
<param name="duration_recovery" value="20"/> <!-- How long the recovery behavior is allowed to execute, if this time (in seconds) is over, send "FINISHED" to the task planning and stop the recovery-->
</node>
 
<!-- Start swot_pose_controller service-->
<node pkg="swot_pose_controller" type="swot_pose_controller" name="swot_pose_controller" respawn="true">
</node>

</launch>
